skl2onnx/common/_topology.py

```diff
@@ -1140,7 +1153,12 @@ def _get_main_opset_version(model):
     """
     Returns the main opset version.
     """
+    mld = None
     for op in model.opset_import:
         if op.domain == '':
```

Comment:
isn't ai.onnx the official name, is it?

Reply:
It is left blank in the onnx graph. With older version of onnx, it was failing with ai.onnx.